rfd/0014-session-2FA.md

```diff
@@ -392,6 +392,15 @@ values are:
 - `usage:apps` (existing) - only web apps
 - `usage:db` (new) - only database connections
 
+### audit log
+
+All audit events related to session secured with 2FA will include a `With2FA`
```

Comment:
@awly now we are using `mfa` and `2fa` at the same time (see CLI command vs other fields). Let's just stick to MFA everywhere?